docs/api.asciidoc

```diff
@@ -194,6 +193,49 @@ elasticapm.set_transaction_result('SUCCESS')
  * `override`: if `True` (the default), overrides any previously set result.
     If `False`, only sets the result if the result hasn't already been set.
 
+
+[float]
+[[api-get-transaction-id]]
+==== `elasticapm.get_transaction_id()`
+
+Get the id of the current transaction. Example:
+
+[source,python]
+----
+import elasticapm
+
+transaction_id = elasticapm.get_transaction_id()
+----
+
+
+[float]
+[[api-get-trace-id]]
+==== `elasticapm.get_trace_id()`
+
+Get the parent_id of the current transaction's trace. Example:
+
+[source,python]
+----
+import elasticapm
+
+trace_parent_id = elasticapm.get_trace_id()
```

Comment:
nitpicking here, but let's call the variable `trace_id` as well :)

Reply:
Done.